CONTRIBUTING.md

```diff
@@ -24,13 +24,23 @@ Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
 Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)
 ```
 
-To install Node (with npm) and Java locally on macOS, you can use [brew](https://brew.sh/):
+To install Node (with npm) and Java locally on macOS, we recommend [brew](https://brew.sh) with 
+[nvm](https://github.com/nvm-sh/nvm):
 
 ```bash
-brew install node
+brew install nvm
+nvm install $(cat .nvmrc)
 brew cask install java
 ```
 
+The `webdriverio` package also needs a compatible version of Python to build. 3.10.x will work with `webdriverio`'s
+dependency `node-gyp`. [`pyenv`](https://github.com/pyenv/pyenv) can help if you don't have this version:
+
+```bash
+brew install pyenv
```

Comment:
Before we go start documenting how to install Python, would you mind making this PR just for Node.js?

I'm going to take a look at our outdated packages and it's likely we can avoid compilation entirely

i.e. No Python needed

Reply:
That would be an awful lot better.